### PR TITLE
Add black formatter

### PR DESCRIPTION
**Pull request by oxytocinlove**
Install `black` formatter, run it on our python files, and add it to `yarn lint` and `yarn fix` commands

### CODE REVIEW

**Comment by blanchardjeremy** [comment]
Looks like lint is failing on server side.

```
Run yarn lint --max-warnings 0
yarn run v1.22.11
$ eslint src && prettier --check src/locales/en.json *.json && black *.py --diff --max-warnings 0
Checking formatting...
All matched files use Prettier code style!
/bin/sh: 1: black: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 127.
```

I don't know exactly what's needed with the workflow. But it lives in `.github/workflows/ci.yml` ... Maybe a requirements.txt file? Or a manual command to install `black`?

File: package.json [hunk 1]
```diff
@@ -39,8 +39,8 @@
     "build": "REACT_APP_DEPLOY_PRIME_URL=\"$DEPLOY_PRIME_URL\" react-scripts build",
     "test": "react-scripts test",
     "coverage": "react-scripts test --coverage --watchAll=false",
-    "lint": "eslint src && prettier --check src/locales/en.json *.json",
-    "fix": "eslint src --fix && prettier --write src/locales/en.json *.json",
+    "lint": "black *.py --diff && eslint src && prettier --check src/locales/en.json *.json",
```
Comment (beshaya) [inline package.json:42]: Have you checked that this will fail CI if the format isn't correct?

**Review by beshaya — APPROVED** [review]
tyty!

File: package.json (hunk 1 above)
Comment (oxytocinlove) [reply package.json:43]: I've checked that it will fail locally but not in CI

**Comment by beshaya** [comment]
Works for me.

**Pull request merged**